### PR TITLE
Refactoring for Issue 588 and added code changes for issue 827

### PR DESCRIPTION
**Pull request by dim505**

### CODE REVIEW

File: src/components/Pages/ActionsPage/StoryForm.js [hunk 1]
```diff
@@ -272,15 +272,21 @@ class StoryForm extends React.Component {
...
-            formNotification: {
-              icon: "fa fa-check",
-              type: "good",
-              text:
-                "Nicely done! Your story will be reviewed and published as soon as possible. Stay tuned!",
-            },
-          });
-          resetForm();
+			
+			if (this.props?.TriggerSuccessNotification) {
```
Comment (BradHN1) [inline src/components/Pages/ActionsPage/StoryForm.js:276]: @dim505 - still having issues with your tab settings?

Comment (BradHN1) [reply src/components/Pages/ActionsPage/StoryForm.js:276]: also note that for javascript, convention is to use camelCase variable names, except for class names.  So this should be triggerSuccessNotification.

File: src/components/Pages/StoriesPage/StoryFormButtonModal.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+import React, { Component } from "react";
+import Button from "react-bootstrap/Button";
+
+
+//refactored the submit testimonial form so now you can have a modal version of it
+class StoryFormButtonModal extends Component {
+  constructor() {
+    super();
+    this.state = {
+      OpenSuccessNotification: false,
```
Comment (BradHN1) [inline src/components/Pages/StoriesPage/StoryFormButtonModal.js:13]: camelCase

**Comment by BradHN1** [comment]
@dim505 - can you move the buttons so they are more centered and larger, and have the "Close" button next to the Submit.  I couldn't find the close button until I searched for it.  Submit button should be green.  Close button could be red and labelled "Cancel" until the testimonial is submitted.
Screen Shot 2022-01-28 at 8 51 26 AM

**Comment by dim505** [comment]
image
updated UI. Please see commit

File: src/components/Pages/ActionsPage/StoryForm.js (hunk 1 above)
Comment (dim505) [reply src/components/Pages/ActionsPage/StoryForm.js:277]: yeah sorry the tab spacing was set to 4

**Review by BradHN1 — APPROVED** [review]
LGTM

**Pull request merged**